src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25859.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25859 : _IssuesUITest
+	{
+		public override string Issue => "Item spacing not properly applied between items in CollectionView Horizontal LinearItemsLayout";
+
+		public Issue25859(TestDevice device) : base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.Editor)]
+		public void CVHorizontalLinearItemsLayoutItemSpacing()
+		{
+			App.WaitForElement("collectionView");
+
+			VerifyScreenshot();
```

Comment:
Snapshots available in the latest build.

Reply:
Snapshots have been added.

Reply:
The test is failing:
![image](https://github.com/user-attachments/assets/0606e9b6-7c93-4700-856b-5ee6ae9240b9)
(in red the differences with the added reference snapshot)

Reply:
In CI, it seems that the sample uses CV2 handlers when working with CollectionView. Since this issue was resolved in CV1, I have updated it to use CollectionView1. Could you please review and let me know if you have any concerns?